include/deal.II/dofs/dof_tools.h

```diff
@@ -362,9 +362,9 @@ namespace DoFTools
    * Remember using SparsityPattern::compress() after generating the pattern.
    *
    * The actual type of the sparsity pattern may be SparsityPattern,
-   * CompressedSparsityPattern, BlockSparsityPattern,
-   * BlockCompressedSparsityPattern, BlockCompressedSetSparsityPattern,
-   * BlockCompressedSimpleSparsityPattern, or any other class that satisfies
+   * DynamicSparsityPattern, BlockSparsityPattern,
+   * BlockDynamicSparsityPattern, BlockDynamicSetSparsityPattern,
+   * BlockDynamicSimpleSparsityPattern, or any other class that satisfies
```

Comment:
The last two of these don't actually exist -- only `DynamicSparsityPattern` is left, no `*Set*` and `*Simple*` variants any more. Same further down below.